pkg/storageincentives/redistribution/redistribution.go

```diff
@@ -143,12 +143,13 @@ func (c *contract) Reveal(ctx context.Context, storageDepth uint8, reserveCommit
 		return err
 	}
 	request := &transaction.TxRequest{
-		To:          &c.incentivesContractAddress,
-		Data:        callData,
-		GasPrice:    sctx.GetGasPrice(ctx),
-		GasLimit:    sctx.GetGasLimit(ctx),
-		Value:       big.NewInt(0),
-		Description: "reveal transaction",
+		To:                   &c.incentivesContractAddress,
+		Data:                 callData,
+		GasPrice:             sctx.GetGasPrice(ctx),
+		GasLimit:             sctx.GetGasLimit(ctx),
+		MinEstimatedGasLimit: 500_000,
```

Comment:
@mrekucci can we add this to all the other calls in this pkg?

Reply:
Done.